deployment/config.go

```diff
@@ -93,7 +93,8 @@ type Config struct {
 	Report                report.Config
 	// Directory under which the .terraform directory and state files are managed.
 	// It will be created if it does not exist
-	TerraformStateDir string `default:"/var/lib/mattermost-load-test-ng" validate:"notempty"`
+	// ./ltstate is the default value used when `deployer.json`` is configured at all
+	TerraformStateDir string `default:"./ltstate" validate:"notempty"`
```

Comment:
Fine with the change, but is this somehow related to the PR?

Reply:
@agnivade: Not the original goal, but to get to the correct state we wanted, we need it.

Reply:
Ah, understood!